From: Dermot Bradley <dermot_bradley@yahoo.com>
Date: Thu, 19 Nov 2020 22:34 +0000
Subject: [PATCH] cloud-init: Remove oauthlib from requirements.txt

oauthlib is only used by the MAAS datasource. Remove it from the Python
requirements.txt file in order to reduce package dependencies. If someone
wishes to use the MAAS datasource they can simply add the py3-oauthlib
package.

---

diff -aur a/requirements.txt b/requirements.txt
--- a/requirements.txt
+++ b/requirements.txt
@@ -5,7 +5,8 @@
 
 # This one is currently only used by the MAAS datasource and the Event
 # reporting feature when configured to use webhooks.
-oauthlib
+#
+# oauthlib
 
 # This one is currently used only by the CloudSigma and SmartOS datasources.
 # If these datasources are removed, this is no longer needed.
